x-pack/test/functional/apps/lens/persistent_context.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import _ from 'lodash';
+import expect from '@kbn/expect';
+import { FtrProviderContext } from '../../ftr_provider_context';
+
+// eslint-disable-next-line import/no-default-export
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const PageObjects = getPageObjects(['visualize', 'header', 'timePicker']);
+  const browser = getService('browser');
+  const filterBar = getService('filterBar');
+  const appsMenu = getService('appsMenu');
+
+  describe('lens query context', () => {
+    it('should carry over time range and pinned filters to discover', async () => {
+      await PageObjects.visualize.navigateToNewVisualization();
+      await PageObjects.visualize.clickVisType('lens');
+      await PageObjects.timePicker.setAbsoluteRange(
+        'Sep 06, 2015 @ 06:31:44.000',
+        'Sep 18, 2025 @ 06:31:44.000'
+      );
```

Comment:
I assume it's intentional that this range doesn't overlap with the default time range, but I'm not sure why? We do have a `PageObjects.lens.goToTimeRange` helper which wraps this.